frontend/src/components/tasks/createTask.jsx

```diff
@@ -26,7 +23,6 @@ let CreateTask = React.createClass({
         description: '',
         status: this.props.status,
```

Comment:
Don't need this line anymore. All new tasks should only have status = 0 (Sequelize is set to use that as default)